Automattic/wp-calypso
# A4A > Referrals: Initial implementation of automated referrals

Resolves https://github.com/Automattic/jetpack-genesis/issues/335
Resolves https://github.com/Automattic/jetpack-genesis/issues/340
Resolves https://github.com/Automattic/jetpack-genesis/issues/341

## Proposed Changes

This PR implements the initial changes required for the automated referrals. These changes are behind a feature flag.

NOTE: Please ignore the page background for now.

## Testing Instructions

1. Open the A4A live link.
2. Verify that you can see a chevron icon next to the Referrals

<img width="270" alt="Screenshot 2024-05-21 at 7 22 14 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/608ab2df-8de6-4c38-a981-cf4cd7d89358">

3. Click the Referrals > Verify that you can see sub-menu items as shown below. Also, verify the link works as expected.

<img width="491" alt="Screenshot 2024-05-21 at 7 22 24 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/29110bfb-d0b8-4f9a-bf3d-def22c32cc47"> 

4. Verify that the feature flag is enabled only on Dev & Horizon
5. Append the /referrals URL as `/referrals?flags=-a4a-automated-referrals` & verify that the above changes are not reflected and the Referrals page matches with the production. 

## Pre-merge Checklist
